### ISSUE TITLE
[Feature suggestions] Change the default tab 

### ISSUE THREAD

**Issue by toyxyz** [hayden-fr/ComfyUI-Model-Manager#34]
Thanks for the great work!

I have one feature suggestion. 

What if user could specify in the settings which tab want to see first when user launch the model manager?

Currently, when I first launch ComfyUI, I see the Download tab first and have to navigate to the model tab. 

Also, if possible, it would be very useful to add a hotkey to show/hide the model manager.

**Comment by cdb-boop** [comment]
> What if user could specify in the settings which tab want to see first when user launch the model manager?

Yeah it's a paper cut sometimes. It was done to avoid loading images when just downloading models, but currently the UI is being redone so changes are frozen.

> Also, if possible, it would be very useful to add a hotkey to show/hide the model manager.

I agree a hot key would be nice. Do you know what ComfyUI's reserved hotkeys are and what are available for extensions to use?

**Comment by toyxyz** [comment]
> > What if user could specify in the settings which tab want to see first when user launch the model manager?
> 
> Yeah it's a paper cut sometimes. It was done to avoid loading images when just downloading models, but currently the UI is being redone so changes are frozen.
> 
> > Also, if possible, it would be very useful to add a hotkey to show/hide the model manager.
> 
> I agree a hot key would be nice. Do you know what ComfyUI's reserved hotkeys are and what are available for extensions to use?

In the latest version of the ComfyUi frontend, there is a menu to view and modify keybindings. 

https://github.com/Comfy-Org/ComfyUI_frontend

![image](https://github.com/user-attachments/assets/75a19989-5f36-47ac-a077-f538962ddbe2)

**Comment by cdb-boop** [comment]
Perfect. From the image I assume the change would be relatively simple, a matter of registering a "Command ID" and handling the event.
